praw/models/helpers.py

```diff
@@ -39,6 +39,27 @@ def create(self, title, description=None, nsfw=False, resources=None):
             'description': description, 'nsfw': nsfw, 'resources': resources,
             'title': title})
 
+    def now(self):
+        """Get the currently featured live thread.
+
+        :returns: The :class:`.LiveThread` object, or ``None`` if there is
+            no currently featured live thread.
+
+        Usage:
+
+        .. code-block:: python
+
+        thread = reddit.live.now()  # LiveThread object or None
+
+        """
+        # We shouldn't call reddit.get() here
+        # because objectify() doesn't accept '' or None
```

Comment:
> Anyway, Is there something I should do for coverage decreasing?

When you update the code to use `reddit.get` the coverage will be back to 100% as you will be testing the new case, and the existing case for the objector already exists.